dev/benchmarks/macrobenchmarks/macos/Podfile

```diff
@@ -9,71 +9,32 @@ project 'Runner', {
   'Release' => :release,
 }
 
-def parse_KV_file(file, separator='=')
-  file_abs_path = File.expand_path(file)
-  if !File.exists? file_abs_path
-    return [];
+def flutter_root
+  generated_xcode_build_settings_path = File.expand_path(File.join('..', 'Flutter', 'ephemeral', 'Flutter-Generated.xcconfig'), __FILE__)
+  unless File.exist?(generated_xcode_build_settings_path)
+    raise "#{generated_xcode_build_settings_path} must exist. If you're running pod install manually, make sure flutter pub get is executed first"
```

Comment:
Nit: "make sure 'flutter pub get' is executed first" (to make parsing out the command part easier)

Reply:
<img width="585" alt="Screen Shot 2020-12-17 at 10 34 25 AM" src="https://user-images.githubusercontent.com/682784/102528469-77204980-4053-11eb-918a-f30f5bb93959.png">